source/elm-live.js

```diff
@@ -194,9 +194,18 @@ ${indent(String(elmMake.error), 2)}
   };
 
   // Watch Elm files
-  const watcher = chokidar.watch('**/*.elm', { ignoreInitial: true, followSymlinks: false });
+  const watcher = chokidar.watch('.', {
```

Comment:
Hey @Natim, can you explain what you mean saying that the pattern matching doesn’t work? Do you mean that `**/*.elm` picks up files that it shouldn’t? Or that it picks too many files?